UPGRADE-3.0.md

```diff
@@ -595,6 +595,24 @@ UPGRADE FROM 2.x to 3.0
       }
    }
    ```
+   
+   If you're using `PUT` requests you need to pass the request method to the form:
```

Comment:
This is not exclusive for `PUT` requests. We should reword this a bit like this:

> If the form is submitted with a different request method than `POST`, you need to configure this in the form:

Reply:
Sounds good to me, that's why I added `$request->getMethod()` in the first place. Do we want to keep the `PUT` as example code? Make sense to me.

Reply:
I think we can keep the added example below as it is (which is good) and just reword this sentence.

Reply:
I agree with @xabbuh, @fnagel can you do the changes?

Reply:
@fabpot @xabbuh Changed!